interface.js

```diff
@@ -0,0 +1,75 @@
+$(document).ready(function() {
+  var match;
+
+  new_match();
+
+  $('#score0').on('click', function() { play(0); })
```

Comment:
We could make this more DRY by assigning a class to the score buttons and doing something like: 

```
for (var i = 0; i < 10; i++) {
  $('.score-button')[i].on('click', function() { play(i); }
}
```